.circleci/config.yml

```diff
@@ -309,7 +313,8 @@ jobs:
       - run:
           name: Set php.ini defaults
           command: printf "date.timezone=UTC\nmemory_limit=-1\n" | sudo tee -a /usr/local/etc/php/php.ini
-      - run: composer update
+      - <<: *STEP_COMPOSER_UPDATE
+      - <<: *STEP_COMPOSER_TESTS_UPDATE
```

Comment:
At first glance I though this was wrong and shouldn't be here, but looking at it again I think this is just the equivalent of `composer update` from what we were doing before, right? It's not updating each integration suite?

Reply:
Correct. This updates the newly created `tests/composer.json` project. Individual web apps relying on composer are updated in respective `test_web_x_y` target.